lua/net.lua

```diff
@@ -25,6 +27,7 @@ return function (gatekeeper_server)
 	local front_iface = gatekeeper.c.get_if_front(net_conf)
 	front_iface.arp_cache_timeout_sec = front_arp_cache_timeout_sec
 	front_iface.nd_cache_timeout_sec = front_nd_cache_timeout_sec
+	front_iface.bonding_mode = front_bonding_mode
```

Comment:
I think we should try to keep all of the code that the user can change in one place, so that they don't need to change anything below the `Code below this point should not need to be changed.` warning.